.deepsource.toml

```
@@ -0,0 +1,7 @@
+version = 1
+
```

We could probably add the following (even though I am not sure if the suggested pattern is ok):
```suggestion

test_patterns = [
  "**/*_test.go"
]
```